skills/apple/swiftui-whats-new-27/references/async-image.md

```
@@ -0,0 +1,68 @@
+# AsyncImage
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)